src/Mod/Cloud/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+add_compile_options("-DBUILD_CLOUD")

```

Comment:
Still required?

Reply:
I just removed it, you was right it is conditionally define at the upper level.